Form/EventListener/ResizeFormListener.php

```diff
@@ -63,8 +63,8 @@ public static function getSubscribedEvents()
     {
         return array(
             FormEvents::PRE_SET_DATA    => 'preSetData',
-            FormEvents::PRE_BIND        => 'preBind',
-            FormEvents::BIND            => 'onBind',
+            FormEvents::PRE_SUBMIT      => 'preSubmit',
```

Comment:
We can also break the code and just add a note and raise an exception if thoses methods are called.